src/Uno.UWP/Networking/Connectivity/Internal/NetworkReachability.iOS.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Adapted from https://github.com/xamarin/Essentials/blob/master/Xamarin.Essentials/Connectivity/Connectivity.ios.tvos.reachability.cs
+#if __IOS__ || __MACOS__
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using SystemConfiguration;
+using CoreFoundation;
+
+namespace Uno.Networking.Connectivity.Internal
+{
+	internal enum NetworkStatus
+	{
+		NotReachable,
+		ReachableViaCarrierDataNetwork,
+		ReachableViaWiFiNetwork
+	}
+
+	internal static class Reachability
+	{
+		internal const string HostName = "www.microsoft.com";
```

Comment:
Should this be configureable?

Reply:
Good idea! Will add some configuration point for this

Reply:
@carldebilly Where should I put this configuration? I thought about `FeatureConfiguration` class, but that is in Uno.UI, so it is not accessible. Would it be appropriate to create a iOS specific public class `Uno.Networking.Connectivity.NetworkInformation` wich a single property like `ReachabilityHostName`?

Reply:
Anyway, if it is for iOS, why test microsoft.com and not e.g. apple.com?

Reply:
@pkar70 I trust Azure stability more than whatever Apple uses to host their site 😂

Reply:
Maybe you are right about stability, but I think iOS code should use some iOS-related host.  This is Apple world, so there is no reason why Apple devices should mass-ping Microsoft :)
Maybe use Apple Store?

Reply:
I think making it configurable is probably the best solution, as we can let the user decide 👍

Reply:
@MartinZikmund You can put it here https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/WinRTFeatureConfiguration.cs

Reply:
Done :-)